src/doc/trpl/ffi.md

```diff
@@ -85,8 +101,11 @@ the allocated memory. The length is less than or equal to the capacity.
 # #![feature(libc)]
 # extern crate libc;
 # use libc::{c_int, size_t};
+# 
```

Comment:
we don't usually add extra lines like this here, as they're not displayed anyway, and just take up space